views/js/qtiCreator/editor/gridEditor/droppable.js

```diff
@@ -262,7 +262,7 @@ define([
             }, 50)
         ).on('mousemove.gridEdit.gridDragDrop',
             _.throttle(function(e){ // '[class^="col-"]:not(.new-col), [class*=" col-"]:not(.new-col)
-                var goingTo = e.relatedTarget|| e.toElement; //browser compatibility
+                var goingTo = e.relatedTarget || e.toElement || e.target; //browser compatibility
```

Comment:
Do we really need to check these 3? Would not be enough to have only e.target? If yes, don't we need these in lines: 308 and 241 also?